resources/scss/modules/_shortcuts.scss

```diff
@@ -1,34 +1,36 @@
-.shortcuts>section {
-    margin-bottom: $spacing-big;
-}
+.operator {
+    padding: $spacing-base 0;
 
-.shortcut {
-    box-sizing: border-box;
+    +.operator {
+        border-top: 2px solid $base-border-color;
+    }
 
-    >div {
-        margin-bottom: $spacing-micro;
-        display: flex;
-        align-items: center;
+    >h5 {
+        margin-top: 0;
+    }
+}
 
-        >:first-child {
-            flex: 1;
-        }
+.shortcuts {
+    border-collapse: separate;
+    border-spacing: $spacing-small $spacing-tiny;
+    margin: 0 (-$spacing-small) (-$spacing-tiny);
 
-        >:last-child {
-            flex: 1;
+    >tbody>tr {
+        >td {
+            padding: 0;
+            margin: 0;
+        }
+        >td:first-child {
+            width: 0;
+            white-space: nowrap;
         }
-    }
-
-    img {
-        margin-bottom: $spacing-small;
     }
 }
 
-.keystroke {
-    code {
-        border: 2px solid $color-default-text;
-        padding: $spacing-micro / 2 $spacing-micro;
-        border-radius: 4px;
-        display: inline-block;
-    }
+.keystroke>kbd {
+    border: 2px solid $color-default-text;
+    padding: 2px 8px;
```

Comment:
For padding, could you use variables? `$spacing-micro` is 8px. Round radiuses are also set by $round-radius - 8px right now.
See _variables.scss for all variables.

Reply:
I'm not sure whether `$spacing-micro/4` is really better than `2px`, but yes, I will do this.